Week05/lesson10/homework/cash_register_starter_code/js/index.js

```diff
@@ -0,0 +1,27 @@
+// Initial total is 0
+// User types in an amount
+// User presses 'return' key to submit new value
+// Cash registrer shows every new value with $ sign and two decimals as a new line on the receipt
+// Calculates new incremental total accordingly with $ sign and two decimals and shows the result under TOTAL
+
+var total = 0;
+
+function sumItAll (event) {
+
+  event.preventDefault();
+
+  amount = $("#newEntry").val();
+
+  amount = parseFloat(amount);
```

Comment:
What happens if the `val()` is not a number? How can you prepare for that situation?